e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -67,13 +73,91 @@ context('When fee record correction feature flag is enabled', () => {
       cy.assertText(premiumPaymentsTab.premiumPaymentsTable.error(), 'Error: Select a record to create a record correction request');
     });
 
-    it('should be able to initiate record correction request', () => {
+    it('should be able to initiate a record correction request', () => {
       premiumPaymentsTab.premiumPaymentsTable.checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status).click();
 
       premiumPaymentsTab.createRecordCorrectionRequestButton().should('exist');
       premiumPaymentsTab.createRecordCorrectionRequestButton().click();
 
       cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}`));
     });
+
+    context('when the user navigates to the "create record correction request" page', () => {
+      const { mainHeading } = pages.createFeeRecordCorrectionRequestPage;
+      const { feeRecordSummary } = partials;
+
+      beforeEach(() => {
+        premiumPaymentsTab.premiumPaymentsTable
+          .checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status)
+          .click();
+
+        premiumPaymentsTab.createRecordCorrectionRequestButton().should('exist');
+        premiumPaymentsTab.createRecordCorrectionRequestButton().click();
+      });
+
+      it('should render the create record correction request page with selected fee record details', () => {
+        cy.assertText(mainHeading(), 'Record correction request');
+
+        feeRecordSummary.container().should('exist');
+        cy.assertText(feeRecordSummary.facilityId(), feeRecordAtToDoStatus.facilityId);
+        cy.assertText(feeRecordSummary.exporter(), feeRecordAtToDoStatus.exporter);
+        cy.assertText(feeRecordSummary.requestedBy(), `${USERS.PDC_RECONCILE.firstName} ${USERS.PDC_RECONCILE.lastName}`);
+      });
+    });
+
+    context('when the user submits the "create record correction request" form with invalid values', () => {
```

Comment:
I'm not familiar with the design/functionality - do we have/need a test for submitting an empty form?

Reply:
I've refactored these tests a little to try and make things a bit simpler when we add more stages to the journey (upcoming tickets).

This has now been moved to the new error cases file, I've also added a test for an empty form as suggested - thanks!

Reply:
For a bit more context, the form validation requirements are:
- 1 or more reasons checked
- Between 1 and 500 characters of additional info

These are tested in unit tests, but I've now got e2e tests (in this PR) for: checking multiple reasons and having valid additional info input, no checked reasons and no additional info input, more additional info characters than allowed